sockeye/init_embedding.py

```diff
@@ -28,8 +29,11 @@
 Optional arguments:
 
     --embeddings, -e
```

Comment:
Now that this change has been generalized to any kind of weight it might make sense to rename the argument (and maybe also the module) to `--weights` or `--weight-files`. What do you think?